csrankings.csv

```diff
@@ -2998,6 +2998,7 @@ Chee-Yong Chan,National University of Singapore,https://www.comp.nus.edu.sg/~cha
 Cheeha Kim,POSTECH,http://phome.postech.ac.kr/user/indexSub.action?codyMenuSeq=2021040&siteId=citeng&menuUIType=top,1UlTiEMAAAAJ
 Chein-I Chang,Univ. of Maryland - Baltimore County,http://www.csee.umbc.edu/people/faculty/chein-i-chang,03Bww5gAAAAJ
 Chellu Chandra Sekhar,IIT Madras,https://www.iitm.ac.in/info/fac/cchandra,rO5ZgW4AAAAJ
+Chelsea Finn,Stanford University,http://ai.stanford.edu/~cbfinn/,vfPE6hgAAAAJ

```

Comment:
Google AI.

Reply:
I know she is 20% at Google since I also work at Google though I don't know what kind of evidence I can provide. What kind of evidence is expected?

Reply:
The easiest thing would be for her to post a note here indicating she is >= 75% at Stanford (over the entire academic year).

Reply:
I can confirm that I'm >75% at Stanford.